WordPress/src/main/java/org/wordpress/android/ui/reader/subfilter/viewholders/SubfilterListItemViewHolder.kt

```diff
@@ -0,0 +1,35 @@
+package org.wordpress.android.ui.reader.subfilter.viewholders
+
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.ImageView
+import android.widget.TextView
+import androidx.annotation.LayoutRes
+import androidx.recyclerview.widget.RecyclerView.ViewHolder
+import org.wordpress.android.R
+import org.wordpress.android.ui.reader.subfilter.SubfilterListItem
+import org.wordpress.android.ui.utils.UiHelpers
+
+open class SubfilterListItemViewHolder(
+    internal val parent: ViewGroup,
+    @LayoutRes layout: Int
+) : ViewHolder(LayoutInflater.from(parent.context).inflate(layout, parent, false)) {
+    open fun bind(filter: SubfilterListItem, uiHelpers: UiHelpers) {
+        val selectedTick: ImageView? = this.itemView.findViewById(R.id.item_selected)
+        selectedTick?.let {
+            it.visibility = if (filter.isSelected) View.VISIBLE else View.GONE
+        }
+        val itemText: TextView? = this.itemView.findViewById(R.id.item_title)
+        itemText?.let {
+            if (filter.isSelected) {
+                @Suppress("DEPRECATION")
+                it.setTextColor(parent.resources.getColor(R.color.primary))
```

Comment:
I think you should use `.setTextColor(ContextCompat.getColor(it, R.color.primary))` instead so you don't have to suppress the warning.

Reply:
Better, thanks for pointing out! 😊Changed here fed0e186b3eb4f9be44740543af84242f01b2cb3